src/main/java/com/ardaslegends/repository/claimbuild/ClaimbuildRepositoryImpl.java

```diff
@@ -63,5 +65,23 @@ public List<ClaimBuild> findClaimBuildsByNames(String[] names) {
         return fetchedClaimbuilds;
     }
 
+    @Override
+    public List<ClaimBuild> findClaimBuildsByFaction(Faction faction) {
```

Comment:
same here, should be called query instead of find just fyi

Reply:
Done